app/helpers/request_info_helper.rb

```diff
@@ -1,9 +1,81 @@
 module RequestInfoHelper
   private
 
+  REQUEST_INFO_TAB_TYPES = {
+    :requester   => N_('Requester'),
+    :purpose     => N_('Purpose'),
+    :catalog     => N_('Catalog'),
+    :environment => N_('Environment'),
+    :properties  => N_('Properties'),
+    :volumes     => N_('Volumes'),
+    :customize   => N_('Customize'),
+    :schedule    => N_('Schedule'),
+    :hardware    => N_('Hardware'),
+    :network     => N_('Network'),
+  }.freeze
+
+  PROV_FIELD_LABLES = {
```

Comment:
@Fryguy , I am not sure if this is the right way to do it. I didn't expect these many items when I started.

Reply:
@Fryguy, all labels are mentioned in [manageiq-providers-openstack/content/miq_dialogs/miq_provision_openstack_dialogs_template.yaml](https://github.com/ManageIQ/manageiq-providers-openstack/blob/6b7d2a11fbd27fa8eba95cece26ae3f0de23df97/content/miq_dialogs/miq_provision_openstack_dialogs_template.yaml)

Reply:
changes may occur in https://github.com/ManageIQ/manageiq/blob/master/config/locale_task_config.yaml